flink-python/pyflink/ml/tests/__init__.py

```diff
@@ -0,0 +1,58 @@
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+import glob
+import os
+import unittest
+
+from pyflink.testing.test_case_utils import PyFlinkTestCase
+from pyflink.find_flink_home import _find_flink_source_root
+
+
+class MLTestCase(PyFlinkTestCase):
+    """
+    Base class for testing ML.
+    """
+
+    _inited = False
+
+    @staticmethod
+    def _ensure_path(pattern):
```

Comment:
Should this method be in `pyflink.testing.test_case_util`?

Reply:
Yes, it would be better.